src/main/java/org/folio/service/orders/flows/update/unopen/UnOpenCompositeOrderManager.java

```diff
@@ -113,29 +127,49 @@ private Future<Void> processInventory(CompositePoLine compPOL, RequestContext re
           logger.debug(String.format("Pieces were removed : %s", deletedIds));
         }
       })
-        .mapEmpty();
+      .mapEmpty();
     } else if (PoLineCommonUtil.isItemsUpdateRequired(compPOL)) {
-      return processInventoryWithItems(compPOL, requestContext);
+        if (compPOL.getCheckinItems()) {
+          return (deleteHoldings) ? processInventoryOnlyWithHolding(compPOL, requestContext) : Future.succeededFuture();
+        } else {
+          return (deleteHoldings) ? processInventoryWithItems(compPOL, requestContext) : processInventoryOnlyWithItems(compPOL, requestContext);
```

Comment:
I think we need to rename this methods: processInventoryHoldingsWithItems

Reply:
fixed